jobs/config.json

```diff
@@ -11423,7 +11423,6 @@
   "pull-test-infra-testgrid": {
     "args": [
       "bazel",
-      "--",
       "run",
       "//testgrid/prow_verify:prow_verify",
       "--",
```

Comment:
do we need this -- still?

Reply:
it should be https://github.com/kubernetes/test-infra/pull/6302#issuecomment-358206016 right?

Reply:
bad reading of github diff, sorry ><